mock-core-configs/etc/mock/templates/centos-stream.tpl

```diff
@@ -75,6 +75,13 @@ gpgcheck=1
 enabled=1
 gpgkey=file:///usr/share/distribution-gpg-keys/centos/RPM-GPG-KEY-CentOS-Official
 
+[Stream-Devel]
+name=CentOS-Stream - Devel WARNING! FOR BUILDROOT USE ONLY!
```

Comment:
Do you want to keep the large warning here?  Saying "Buildroot only." would be appropriate to me :-).  Otherwise no problem, ... no strong objection, the PR looks good to me.

Reply:
I copied it from [the official repo file](https://git.centos.org/rpms/centos-release/blob/c8/f/SOURCES/CentOS-Devel.repo), so I think it's appropriate to leave as-is.